services/payment/swagger.yaml

```diff
@@ -0,0 +1,132 @@
+swagger: '2.0'
```

Comment:
Move this to `swaggerConfig.js` in `src` and update routes with annotations

Reply:
sure sounds good, thanks!